game-core/src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -229,9 +229,6 @@ private void selectUnitsToRemove(
       new DefaultMapSelectionListener() {
         @Override
         public void territorySelected(final Territory territory, final MouseDetails md) {
-          if (territory == null) {
```

Comment:
Are you sure this is non-nullable, or were we missing the @Nullable annotation?

Side-note on a larger question beyond the scope of this update: I wonder if we would have been better off marking things both as '@Nullable' and/or as "@Nonnull" until we got everything marked. Once everything is marked, then we would be safe to know that a missing '@Nullable' truly means '@Nonnull'

Reply:
bump @RoiEXLab

Reply:
Thanks for the reminder.
I did verify this is non-nullable.
I thought about adding a non-null annotation as well, but wasn't really sure about that